frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -12,16 +12,16 @@ const formatCount = (count: number) => {
   return pluralizeWithNotation(count / 1000000, 'M', 'Object');
 };
 
-const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status }) => (
+const BucketsRowStatus: React.FC<BucketsRowStatusProps> = React.memo(({ status, link }) => (
   <div className="nb-buckets-card__row-status-item">
     {_.isNil(status) ? (
       <span className="nb-buckets-card__row-subtitle">Unavailable</span>
     ) : Number(status) > 0 ? (
       <React.Fragment>
-        <div>
-          <ArrowCircleDownIcon />
-        </div>
-        <div className="nb-buckets-card__row-status-item-text">{status}</div>
+        <a href={link} style={{ textDecoration: 'none' }}>
```

Comment:
Can we define class in scss and use it instead of style?

Reply:
same what if the link is empty string?

Reply:
@cloudbehl its getting override by other css rules . Its done here as well https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/inventory-card/inventory-item.tsx.

Reply:
if the link is empty it takes back to '/dashboards' , this is the default case I guess for all empty links in the console

Reply:
ok, my point is if we can't make(due to any case) the label then we should not add the link property itself. @rawagner thoughts?

Reply:
we should not refresh it unnecessarily when we know the link is not there.

Reply:
yes, i agree with @cloudbehl - do not render link if we dont have one

Reply:
okay, we need to make it non clickable , if we do not receive a proper link.
I also feel the same, redirecting back to the dashboards does not sounds good.

Reply:
also FYI later these links will become internal (probably in 4.3), once we get details page for ob/obc.